wireapp/wire-android-reloaded
# feat: start new a call when there is already an active call - PART 2 (AR-1664)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1664" title="AR-1664" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1664</a>  User tries to make a call, when in a call already, calling UI appears
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Description

A follow up PR of #750

It aims to handle the case of starting a new call when we are already in an active call

Needs releases with:


### Testing

#### How to Test

When you are  in a call, try to make another call, you should see an dialog asking you to confirm that you are going to end the current call and start a new one

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
